hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -36,8 +36,8 @@
 @InterfaceAudience.Public
 @InterfaceStability.Unstable
 public final class OzoneConfigKeys {
-  public static final String DFS_CONTAINER_IPC_PORT =
-      "dfs.container.ipc";
+  public static final String OZONE_CONTAINER_IPC_PORT =
+      "ozone.container.ipc.port";
```

Comment:
To reduce the size of the patch, I suggest renaming items in two steps:

1. config keys (`dfs.container.ipc` to `ozone.container.ipc.port`)
2. java constants (`DFS_CONTAINER_IPC_PORT` to `OZONE_CONTAINER_IPC_PORT`)

Reply:
Added backward compatibility, made all the necessary fixes and got a clean CI run.
I'll keep in my mind about reducing the size of the patch in the coming PRs.